frontend/src/pages/FlightPage/FlightPage.tsx

```diff
@@ -30,12 +44,16 @@ export default function FlightPage({ data, fetchAllFlights }: Readonly<FlightPag
                 }}
             >
                 <AddCircleIcon
-                    sx={{ fontSize: "35px", cursor: "pointer" }}
+                    sx={{fontSize: "35px", cursor: "pointer"}}
                     onClick={handleClick}
                 />
             </Box>
-            <FlightFilter />
-            <FlightList data={data} fetchAllFlights={fetchAllFlights} />
+            <section style={{border: "1px solid #523d35", borderRadius: "2px", alignContent: "center"}}>
+                <button onClick={() => setShowFilter(!showFilter)} className={"show-filter-btn"}>{!showFilter ? 'Show Filters' : 'Close Filters'}</button>
+                {showFilter && <FlightFilter selectedFilter={selectedFilter} setSelectedFilter={setSelectedFilter}/>}
+            </section>
+            <FlightList data={filteredFlightData} fetchAllFlights={fetchAllFlights}/>
+            {filteredFlightData.length == 0 && <h5>No Flights found</h5>}
```

Comment:
I'm not sure, but wouldn't it make sense to to only show the component FlightList if the length is not 0? So having the component in the conditional statement as the else option
```suggestion
            {filteredFlightData.length == 0 ? <h5>No Flights found</h5> : <FlightList data={filteredFlightData} fetchAllFlights={fetchAllFlights}/>}
```

Reply:
Thank you for your review! :) I have corrected them and the changes work well! :)